framework/sessions/__init__.py

```diff
@@ -94,9 +94,16 @@ def set_previous_url(url=None):
     url = url or request.path
     if any([rule(url) for rule in settings.SESSION_HISTORY_IGNORE_RULES]):
         return
+
```

Comment:
`goback()` was indeed never used. I did an exhaustive site-wide search, and the `Session.data['history']` list was also never used. I've removed `goback()`, `set_previous_url()`, the `history` key in the `session.data` dict, and the `after_request` handler.

In total, your comment justified my removing a new 58 lines of OSF code - you are officially my favorite person today. :joy: